### ISSUE TITLE
Pandoc 3.2.1 introduced new LaTex macro \pandocbounded

### ISSUE THREAD

**Issue by sonic-martin** [Wandmalfarbe/pandoc-latex-template#391]
**Explain the problem.**

Pandoc introduced in [release 3.2.1](https://pandoc.org/releases.html) new LaTex macro `\pandocbounded` which custom templates needs to provide:

>LaTeX writer:
>
>New method for ensuring images don’t overflow ([#9660](https://github.com/jgm/pandoc/issues/9660)). Previously we relied on graphicx internals and made global changes to Gin to force images to be resized if they exceed textwidth. This approach is brittle and caused problems with \includesvg (see [#9660](https://github.com/jgm/pandoc/issues/9660)). The new approach uses a new macro \pandocbounded that is now defined in the LaTeX template. (Thanks here to Falk Hanisch in https://github.com/mrpiggi/svg/issues/60.) The LaTeX writer has been changed to enclose \includegraphics and \includesvg commands in this macro when they don’t explicitly specify a width or height. In addition, the writer now adds keepaspectratio to the \includegraphics or \includesvg options if height is specified without width, or vice versa. Previously, this was set in the preamble as a global option. Users should attend to the following compatibility issues:
> If custom templates are used with the new LaTeX writer, they will have to be updated to include the new \pandocbounded macro, or an error will be raised because of the undefined macro.

```shell
pandoc readme.md -o doc.pdf --template=eisvogel
```

with readme.md:
```text
# Reproducer
![image](picture.png "Random test image")
```

**Pandoc version?**

Pandoc 3.2.1 using the pandoc:extra docker image.

**Reproducer**

See https://github.com/sonic-martin/pandoc3.2.1-eisvogel-reproducer

**Comment by iandol** [comment]
https://github.com/jgm/pandoc-templates/commit/6c0e7b0a4f990debcd38b5c3bd8599193ae8f5a6#diff-f218051b4ca8f740a9f585a149101d4a3025037c568b391b5216edf7b14cfadc

**Comment by jvsteiner** [comment]
Bump - I am stuck on on pandoc 3.2 because I use eisvogel extensively (thanks for the awesomeness BTW)

**Comment by ingcsmoreno** [comment]
Hi, did you find any workarounds in the meantime? Facing this right now.

**Comment by iandol** [comment]
Add the code changes yourself, the diff is quite small (see my post above).

**Comment by iandol** [comment]
Here is my modified eisvogel template that works with Pandoc 3.2.1 (but note I made other changes to support academic metadata etc.):

https://github.com/iandol/dotpandoc/blob/master/templates/eisvogel.latex#L375

changes around line 375

**Comment by jvsteiner** [comment]
I did it myself as well, however, I think it would be better to incorporate them into the repository.  I will work up a PR later today.

**Comment by jvsteiner** [comment]
https://github.com/Wandmalfarbe/pandoc-latex-template/pull/392

**Comment by sonic-martin** [comment]
I looked into the build process of the docker container images, and when your PR https://github.com/Wandmalfarbe/pandoc-latex-template/pull/392 would be merged, then in the next build it would be included in the pandoc docker files as well.

See https://github.com/pandoc/dockerfiles/blob/8ba91aaf8b81b44351044f706d557b97ad39470d/alpine/Dockerfile#L184

**Comment by jvsteiner** [comment]
I think the next line, with the version number will need to be updated though.

**Comment by chdemko** [comment]
See #192

**Comment by class101** [comment]
Thank you @iandol for pointing out the little modification and all for working on this awesome template, it produces PDFs over my expectation, well done.

**Comment by fujohnwang** [comment]
block with same error。。。

```
Error producing PDF.
! Undefined control sequence.
l.220 \pandocbounded

```

```
$ pandoc --version
pandoc 3.3
Features: +server +lua
Scripting engine: Lua 5.4
User data directory: /Users/fq/.local/share/pandoc
Copyright (C) 2006-2024 John MacFarlane. Web: https://pandoc.org
This is free software; see the source for copying conditions. There is no
warranty, not even for merchantability or fitness for a particular purpose.
```

**Comment by appomsk** [comment]
When exporting jupyter notebook in pdf, pandoc 3.3
```
...
! Undefined control sequence.
l.442 \pandocbounded
                    {\includegraphics[keepaspectratio]{image.png}}
? 
! Emergency stop.
```

**Comment by fujohnwang** [comment]
> block with same error。。。
> 
> ```
> Error producing PDF.
> ! Undefined control sequence.
> l.220 \pandocbounded
> ```
> 
> ```
> $ pandoc --version
> pandoc 3.3
> Features: +server +lua
> Scripting engine: Lua 5.4
> User data directory: /Users/fq/.local/share/pandoc
> Copyright (C) 2006-2024 John MacFarlane. Web: https://pandoc.org
> This is free software; see the source for copying conditions. There is no
> warranty, not even for merchantability or fitness for a particular purpose.
> ```

add default defintion of pandocbounded to my pdf template, everything goes well: 

```latex
\newcommand{\pandocbounded}[1]{#1}
```

**Comment by simondueckert** [comment]
I have the same error

`Error producing PDF.
! Undefined control sequence.`

Since I work with the pandoc/extra docker image I cannot modify latex templates. Anything else I can do to fix the problem in my setting?

**Comment by sonic-martin** [comment]
> I have the same error
> 
> `Error producing PDF. ! Undefined control sequence.`
> 
> Since I work with the pandoc/extra docker image I cannot modify latex templates. Anything else I can do to fix the problem in my setting?

Based on the `pandoc/extra` container image I built an own image adding the `eisvogel.tex` file from https://github.com/Wandmalfarbe/pandoc-latex-template/pull/392.

```Dockerfile
FROM pandoc/extra:3.4.0
COPY eisvogel.latex /usr/local/share/pandoc/templates
```

But see https://github.com/Wandmalfarbe/pandoc-latex-template/pull/392, the maintainer of the Eisvogel template will shortly release a new fixed version.

**Comment by Wandmalfarbe** [comment]
This issue should be fixed in [version 2.5.0](https://github.com/Wandmalfarbe/pandoc-latex-template/releases/tag/v2.5.0). Thank you @jvsteiner for providing a Pull Request.
